Makefile

```diff
@@ -85,30 +85,30 @@ help:/
 
 .PHONY: build
 ## Build server and client.
-build: prebuild-check $(BINARY_SERVER_BIN) $(BINARY_CLIENT_BIN) # do the build
+build: prebuild-check generate $(BINARY_SERVER_BIN) $(BINARY_CLIENT_BIN) # do the build
 
-$(BINARY_SERVER_BIN): prebuild-check $(SOURCES)
+$(BINARY_SERVER_BIN): $(SOURCES)
 ifeq ($(OS),Windows_NT)
 	go build -v ${LDFLAGS} -o "$(shell cygpath --windows '$(BINARY_SERVER_BIN)')"
 else
 	go build -v ${LDFLAGS} -o ${BINARY_SERVER_BIN}
 endif
 
-$(BINARY_CLIENT_BIN): prebuild-check $(SOURCES)
+$(BINARY_CLIENT_BIN): $(SOURCES)
 ifeq ($(OS),Windows_NT)
 	cd ${CLIENT_DIR}/ && go build -v ${LDFLAGS} -o "$(shell cygpath --windows '$(BINARY_CLIENT_BIN)')"
 else
 	cd ${CLIENT_DIR}/ && go build -v -o ${BINARY_CLIENT_BIN}
 endif
 
 # These are binary tools from our vendored packages
-$(GOAGEN_BIN): prebuild-check
+$(GOAGEN_BIN):
```

Comment:
Since the only requirement for those targets to work is to have Go installed and the dependencies fetched, this looks good.